### PR TITLE
Getting rid of Name(s) deleted hack

### PR DESCRIPTION
**Pull request by alaaalzibda**

### CODE REVIEW

File: app/src/main/res/values/edit_name_list_strings.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
           You currently have %d copies of this name.
      </string>
 
-    <string name="names_deleted">Name(s) deleted.</string>
+    <string name="names_deleted">%d instances of \"%s\" deleted from the list</string>
```
Comment (Gear61) [inline app/src/main/res/values/edit_name_list_strings.xml]:
```suggestion
    <string name="names_deleted">%d instances of \"%s\" deleted from the list.</string>
```

**Review by Gear61 — CHANGES_REQUESTED** [review]
Can you add a screenshot/video demo of both cases as well?

**Comment by alaaalzibda** [comment]
Screenshot_20200213-095234
Screenshot_20200213-095248

File: app/src/main/res/values/edit_name_list_strings.xml (hunk 1 above)
Comment (alaaalzibda) [reply app/src/main/res/values/edit_name_list_strings.xml]: Sure!

Comment (Gear61) [reply app/src/main/res/values/edit_name_list_strings.xml]: Can you add the period at the end? We'll be good to go after that.

**Review by Gear61 — APPROVED** [review]
Awesome!

**Pull request merged**